<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pastas</class>
 <widget class="QDialog" name="Pastas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Metros cuadrados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regresar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Regresar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_metros2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_agregar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+ Agregar Lechada</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_provedor">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Provedor</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Material">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Composición de Pasta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Proporción Aglutinante1 - Aglutinante2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_propMat1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_propMat2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_propMat3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Porcentaje de desperdicio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>mm. de espesor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_porcentajeDesperdicio">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mmEspesor">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
